### ISSUE TITLE
will 1st-5th groups C3D weights modified via back-propagation?

### ISSUE THREAD

**Issue by rambleramble** [VisionLearningGroup/R-C3D#52]
It seems that the [1st-5th](https://github.com/VisionLearningGroup/R-C3D/blob/master/experiments/thumos14/train.prototxt) group won't be changing weights during the training since there is a pre-trained model in your description below:
"The C3D model weight pretrained on Sports1M and finetuned on ActivityNet dataset"

Is this a correct understanding?
Can you provide a more comprehensive expalantion about "end-to-end", in terms of training via back-propagation? For instance, maybe we can use your paper's Fighure 2 as a reference.
Thanks a lot!

**Comment by huijuan88** [comment]
The learning rate at these layers can also be open. But the training time will increase. It is a trade-off.

On Jan 29, 2019, at 09:49, rambleramble <notifications@github.com<mailto:notifications@github.com>> wrote:


It seems that the 1st-5th group won't be changing weights during the training since there is a pre-trained model in your description<https://github.com/VisionLearningGroup/R-C3D/blob/master/experiments/thumos14/train.prototxt>

"The C3D model weight pretrained on Sports1M and finetuned on ActivityNet dataset"

Is this a correct understanding?
Can you provide a more comprehensive expalantion about "end-to-end", in terms of training via back-propagation? For instance, maybe we can use your paper's Fighure 2 as a reference.
Thanks a lot!
